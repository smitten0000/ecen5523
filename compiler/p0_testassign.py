a = 3  
b = 4
c = a + b
a = 2 #{b,c}
d = b + c #{b,c}
