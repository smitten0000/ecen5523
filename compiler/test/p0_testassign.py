a = (3+4+5+6)+5
print a
b = -2+5
print b
a = a + b
print a
c = -5
print c
c = -2
print -c+-b
a = (3+4+5+6)+5
print a
b = -2+5
print b
a = a + b
print a
c = -5
print c
c = -2
print -c+-b
