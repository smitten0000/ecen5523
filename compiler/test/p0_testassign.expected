23
3
26
-5
-1
23
3
26
-5
-1
